data_models/run_tests.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# script for executing Django Unittests within Docker container.
+
+docker build -t models_tests -f Dockerfile.tests .
+
+HOST_IP=$(ifconfig eth0 | grep "inet " | awk -F'[: ]+' '{ print $4 }')
+docker run \
+       --add-host=database:$HOST_IP \
+       --env-file /home/kurt/Development/data_refinery/data_models/environments/test \
+       -i models_tests "$@"
```

Comment:
This `env-file` path does not look like something that would work for someone else's build... unless they also had a username of `kurt`.